backend/controllers/movie.mjs

```diff
@@ -9,12 +9,12 @@ const movieCtrl = Router()
 movieCtrl.get('/', authUser , async (req,res,next) => {
     try{
         const userId = req.body.user._id;
-        const movies = await getUserAsync(userId)
-        .then(user => getMoviesAsync(user.watchList) ).catch(() => null);
```

Comment:
I'm also remove the catch method because you already wrapped the whole function with try catch

Reply:
There are some scope issues with error handling when doing thet, Ill check it out tho